Presentation.Web/app/components/data-processing/data-processing-registration-overview.controller.ts

```diff
@@ -422,6 +422,15 @@
                             .withDataSourceType(Utility.KendoGrid.KendoGridColumnDataSourceType.Date)
                             .withInitialVisibility(false)
                             .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.LastChangedAt)))
+                    .withColumn(builder =>
+                        builder
+                        .withDataSourceName("OversightScheduledInspectionDate")
+                        .withTitle("Scheduled inspection date title")
+                        .withId("scheduledInspectionDate")
+                        .withDataSourceType(Utility.KendoGrid.KendoGridColumnDataSourceType.Date)
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Date)
+                        .withStandardWidth(170)
```

Comment:
make sure the size of the column makes the name fit on one line (use the "reset columns" to see the effects)